Formula/v8.rb

```diff
@@ -62,7 +62,7 @@ class V8 < Formula
 
   resource "v8/third_party/icu" do
     url "https://chromium.googlesource.com/chromium/deps/icu.git",
-        revision: "1fd0dbea04448c3f73fe5cb7599f9472f0f107f1"
+        revision: "50ec7b3822a90d73aa761b21fc941b485a1cb9d6"
```

Comment:
@carlocab [This ref should actually exist](https://chromium.googlesource.com/chromium/deps/icu.git/+/50ec7b3822a90d73aa761b21fc941b485a1cb9d6) :)